types/react-linkify/index.d.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as React from "react";
 
-declare namespace ReactLinkify {
+export namespace ReactLinkify {
```

Comment:
This is incorrect change.. The export of this namespace also exports value space (because of class `ReactLinkify` class declaration in it) but `react-linkify` doesnt export any value names `ReactLinkify`
Ideally you would want to move all the definitions in the namespace outside the namespace (removing the namespace declaration) and export only interface definitions

Reply:
That sounds right. 👍